pool/pool_test.go

```diff
@@ -71,7 +72,11 @@ func teardownDB(db *bolt.DB, dbPath string) error {
 	if err != nil {
 		return err
 	}
-	return nil
+	backup := filepath.Join(filepath.Dir(db.Path()), backupFile)
+	if _, err := os.Stat(backup); os.IsNotExist(err) {
+		return nil
+	}
+	return os.Remove(backup)
```

Comment:
Maybe I am misunderstanding, but this seems dangerous. Why should _test_ code by removing a backup file which is created by _production_ code? 

This could potentially delete an important backup of a production database, which is a very drastic consequence of just running some tests.

Reply:
I think you are misunderstanding. The test code runs from the project directory, the backup created is within the project directory as a result, the backup deletion is clearing files remaining after running tests. This has nothing to do with production. If you notice the back up is created using the path of the db, in production the backup will be created where the db resides per the configuration file, not in the project directory.

Reply:
Understood. My confusion come from them both using the same `backupFile` var, but that is only the file name, and now I notice that the path is different.